figures/S7.supplemental_figure7/scripts/S7.r

```diff
@@ -0,0 +1,288 @@
+    header = TRUE,
+    stringsAsFactors = FALSE
+)
+head(platemap_file)
+
+# merge the combined metadata file with plate map on Metadata_Well
+updated_platemap <- merge(
+    x = platemap_file,
+    y = combined_metadata,
+    by.x = "well_id",
+    by.y = "Metadata_Well",
+    all.x = TRUE
+)
+head(updated_platemap)
+unique(updated_platemap$Metadata_cell_type)
+# replace "" in cell_type with "Blank" in label
+# updated_platemap$Metadata_cell_type[updated_platemap$Metadata_cell_type == NA] <- "Blank"
+# replace NA in cell_type with "Blank" in label
+updated_platemap$Metadata_cell_type[is.na(updated_platemap$Metadata_cell_type)] <- "Blank"
+# replace NA with "Blank" in label
+updated_platemap$label[is.na(updated_platemap$label)] <- "Blank"
+# replace pool with Pool in label
+updated_platemap$label[updated_platemap$label == "pool"] <- "Pool"
+# replace holdout with Holdout in label
+updated_platemap$label[updated_platemap$label == "holdout"] <- "Holdout"
+# replace treatment_holdout with Treatment Holdout in label
+updated_platemap$label[updated_platemap$label == "treatment_holdout"] <- "Treatment Holdout"
+unique(updated_platemap$Metadata_cell_type)
+unique(updated_platemap$label)
+
+# Make the cell type factor ordered
+updated_platemap$Metadata_cell_type <- factor(
+    x = updated_platemap$Metadata_cell_type,
+    levels = c(
+        "Blank",
+        "PBMC",
+        "SH-SY5Y"
+    )
+)
+# make the label factor ordered
+updated_platemap$label <- factor(
+    x = updated_platemap$label,
+    levels = c(
+        'Blank',
+        'Pool',
+        'Holdout',
+        'Treatment Holdout'
+    )
+)
+
+width <- 14
+height <- 14
+options(repr.plot.width = width, repr.plot.height = height, units = "cm")
+# set pallette
+viridis_pal_custom <- viridis::viridis_pal(option = "C")(3)
+
+data_split_plate_map <- (
+    raw_map(
+        data = updated_platemap$label,
+        well = updated_platemap$well_id,
+        plate = 384,
+        size = 14)
+    + theme_dark()
+        + ggplot2::geom_point(
+        aes(shape = updated_platemap$Metadata_cell_type),
+        size = 5
+        )
+        + labs(fill = "Data Split", shape = "Cell Type")
+    # change legend text size for fill
+
+    + guides(shape = guide_legend(override.aes = list(size = 12), nrow = 1))
+    + guides(fill = guide_legend(override.aes = list(size = 12),ncol = 2))
+    + theme(
+        legend.title = element_text(size = 18,hjust = 0.5),
+        legend.text = element_text(size = 16),
+    )
+    + scale_shape_manual(
+        values = c(
+            'Blank' = 0,
+            'PBMC' = 19,
+            'SH-SY5Y' = 8
+        )
+    )
+    + scale_fill_manual(
+    values = c(
+        'Blank' = "grey",
+        'Pool' = viridis_pal_custom[1],
+        'Holdout' = viridis_pal_custom[2],
+        'Treatment Holdout' = viridis_pal_custom[3]
+
+    )
+    )
+    + theme(
+        axis.text.x = element_text(size = 16),
+        axis.text.y = element_text(size = 16)
+    )
+
+)
```

Comment:
Consider adding a few comments to help describe what each section of this does and why.